lib/scriptinterpreter.js

```diff
@@ -3,6 +3,7 @@ var bignum = require('bigi');
 var Script = require('btc-script');
 var cryptoHash = require('crypto-hashing')
 var ecdsa = require('ecdsa')
+var Transaction = require('btc-transaction').Transaction
```

Comment:
It'd probably be worth using `bitcoinjs-lib` over `btc-transaction`,  its probably more maintained?
On another note, @jprichardson thoughts on `bitcoinjs-lib/transaction`?
Should make imports/browserifying cleaner :+1: